lib/pages/Home.dart

```diff
@@ -16,17 +20,61 @@ class _HomeState extends State<Home> {
       body: Center(
         child: Column(
           mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            Text(
-              'Tracking current statistics of confirmed COVID-19 cases and history in Zamboanga City',
-              textAlign: TextAlign.center,
-            ),
-            Text(
-              'Coming soon...',
-            ),
-          ],
+          children: homeChildren(width, height)
```

Comment:
Make cards stick to app bar.

Reply:
Request had been committed.

Reply:
![image](https://user-images.githubusercontent.com/15794235/78972844-49d4e080-7b41-11ea-994e-4d1f557fa9d2.png)